web/src/components/CityContentToolbar.tsx

```diff
@@ -20,7 +20,7 @@ const COPY_TIMEOUT = 3000
 const CityContentToolbar = (props: CityContentToolbarProps) => {
   const { children, openFeedbackModal, iconDirection, hasFeedbackOption = true, hasDivider } = props
   const [linkCopied, setLinkCopied] = useState<boolean>(false)
-  const { t } = useTranslation('categories')
+  const { t } = useTranslation(['categories', 'cityNotCooperating'])
```

Comment:
Optional: This is actually not necessary, you could keep it like before and then still use `t('copyUrl')` and `t('cityNotCooperating:textCopied')`.
You just need to add the namespace in t keys for unspecified namespaces.

Reply:
thx for the hint. didn't know. adjusted